langkit/input_output.py

```diff
@@ -23,9 +24,11 @@ def init(transformer_name: Optional[str]):
 init()
 
 
-@register_metric_udf(col_name="combined")
-def similarity_MiniLM_L6_v2(combined: Tuple[str, str]) -> float:
-    x, y = combined
+@register_dataset_udf(col_names=["prompt", "response"])
+def similarity_MiniLM_L6_v2(text):
```

Comment:
I like the idea of having the name of the model in the column name. The only problem is that we already support changing the model on `init` through passing a `transformer_name`. If the user uses a different model, then we have a misleading name. We should either change the name of the udf if `transformer_name` is present, or drop the model's name altogether.

Reply:
Also, I'd personally like to make it clear that this is a similarity score between the input and output. maybe "input_output_similarity", or "pairwise_similarity" or "io_similarity"? @bernease any suggestions?

Reply:
I think you can if it's a global initialized before the registration happens.

Reply:
Er, are you suggesting the user can call `init()` multiple times with different model names? The UDF can only be registered once unless we come up with an unregistration mechanism...